cns/deviceplugin/server.go

```diff
@@ -0,0 +1,153 @@
+package deviceplugin
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"k8s.io/kubelet/pkg/apis/deviceplugin/v1beta1"
+)
+
+const ErrorChanCapacity = 2
```

Comment:
I'd stick this next to the channel and mute the linter. I think the extra indirection makes it harder to read.

Reply:
updated